Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs

```diff
@@ -2,12 +2,14 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text.Json;
+using Flow.Launcher.Infrastructure.Logger;
```

Comment:
Infrastructure Logger, What's this for?

Reply:
oh mb forget about this is for plugin. Though I think somewhere else is still using Infra? I will check soon

Reply:
Yeah we ref Infra for Enum displaying...Unless we want to get rid of that it seems to ok to ref the static Log?

Reply:
hmm actually used for calling logging, when we redo logging we can move it to Plugin project so there is less/no dependency on the Infrastructure project.